plugins/com.redhat.fabric8analytics.lsp.eclipse.core/src/com/redhat/fabric8analytics/lsp/eclipse/core/RecommenderAPIProvider.java

```diff
@@ -65,7 +74,7 @@ public RecommenderAPIProvider(AnalyticsAuthData analyticsAuthData) {
 	 * @param pomFiles
 	 * @return jobID
 	 */
-	public String requestAnalyses(Map<String, String> files) throws RecommenderAPIException {
+	public String requestAnalyses(Map<String, String> files, Set<IFile> license) throws RecommenderAPIException {
```

Comment:
The license file can be only one so there is no need to use Set<IFile>, use single IFile instead